### PR TITLE
Fix terraform task to work with the new TF 0.12.1

### PR DESCRIPTION
**Pull request by ahmedig**
Sorry for lots of the linting changes.
Basically this fix is to remove the additional arguments when you are running a terraform validate, as it complains about the -input=false argument.

### CODE REVIEW

File: Xpirit-Vsts-Release-Terraform/terraform.ps1 [hunk 1]
```diff
@@ -132,33 +116,33 @@ function Initialize-Terraform
...
         Write-Host "##vso[task.complete result=Failed]"
     }
 }
 
-function Invoke-Terraform
-{
+function Invoke-Terraform {
     $arguments = (Get-VstsInput -Name Arguments -Require) -split '\s+'
 
     $defaultArgs = "-input=false -no-color " + (Get-VstsInput -Name PlanPath)
```
Comment (ahmedig) [inline Xpirit-Vsts-Release-Terraform/terraform.ps1:138]: Thats the whole fix.

**Comment by ahmedig** [comment]
@pgroene appreciate your quick approval, as currently this task doesn't work with the new terraform.

**Comment by pgroene** [comment]
Thank you for making the changes. I did a separate checkin with only the change without the formatting. I released the changes on the marketplace, can you verify it works correctly @ahmedig ?

**Comment by ahmedig** [comment]
Yep, now it works! @pgroene 
Thanks a lot.

`#[command]"terraform" validate -var-file="dev.tfvars" -no-color`

**Pull request closed without merging**